airbyte-integrations/connectors/destination-gcs/src/main/java/io/airbyte/integrations/destination/gcs/GcsDestination.java

```diff
@@ -57,14 +57,14 @@ public AirbyteConnectionStatus check(final JsonNode config) {
 
       return new AirbyteConnectionStatus().withStatus(Status.SUCCEEDED);
     } catch (final AmazonS3Exception e) {
-      LOGGER.error("Exception attempting to access the AWS bucket: {}", e.getMessage());
+      LOGGER.error("Exception attempting to access the Gcs bucket: {}", e.getMessage());
       final String message = getErrorMessage(e.getErrorCode(), 0, e.getMessage(), e);
       AirbyteTraceMessageUtility.emitConfigErrorTrace(e, message);
       return new AirbyteConnectionStatus()
           .withStatus(Status.FAILED)
           .withMessage(message);
     } catch (final Exception e) {
-      LOGGER.error("Exception attempting to access the AWS bucket: {}. Please make sure you account has all of these roles: {}", e.getMessage(),
+      LOGGER.error("Exception attempting to access the Gcs bucket: {}. Please make sure you account has all of these roles: {}", e.getMessage(),
```

Comment:
```suggestion
      LOGGER.error("Exception attempting to access the GCS bucket: {}. Please make sure you account has all of these roles: {}", e.getMessage(),
```

Reply:
In the log message below this it is spelled `Gcs`, is it blocking to update that string too?

Reply:
🙄 not really